index.php

```diff
@@ -21,10 +21,10 @@
 		<meta http-equiv="content-language" content="<? echo $lang; ?>" />
 		<meta name="keywords" content="openstreetmap, openrailwaymap, alexander matheisen, rurseekatze, openlayers, osm, matheisen, orm, eisenbahnkarte, bahnkarte, railmap, railway, railways, eisenbahn, streckenkarte" />
 		<meta name="title" content="OpenRailwayMap" />
-		<meta name="author" content="rurseekatze, Alexander Matheisen" />
+		<meta name="author" content="Unknown" />
 		<meta name="publisher" content="rurseekatze, Alexander Matheisen" />
 		<meta name="revisit-after" content="after 10 days" />
-		<meta name="date" content="2010-01-01" />
+		<meta name="date" content="2015-11-03T00:05:56+0100" />
```

Comment:
It would be better to read this date from the RSS feed. For now I will merge that, but I will improve it later (or keep it and consider this for the redesigned website).